### DISCUSSION TITLE
Look what I found

### DISCUSSION THREAD

**Discussion by sumeshir26** [sumeshir26/TimerX#91] [Ideas]
@im-coder-lg @not-nef https://github.com/martinet101/win32mica

**Comment by lg-nineoneninethreenine** [comment]
That means I can't compile TimerX on my PC. I want to make the move to other OSes, but just can't find the right time + scared to wreck that computer, it's really precious to me.

**Comment by not-nef** [comment]
> @im-coder-lg @not-nef https://github.com/martinet101/win32mica

oooooo nice, who wants to implement it?

**Comment by not-nef** [comment]
> @im-coder-lg @not-nef https://github.com/martinet101/win32mica

YOOOO THIS LOOKS SOO GOOD AND IT DOESNT HAVE ALL OF THE THINGS I HATE ABOUT BLURWINDOW LETSGOOOO AAAAAA

**Comment by not-nef** [comment]
Maybe we can even bring back darkdetect for this!!

**Reply by sumeshir26** [reply]
Yeah!

**Comment by not-nef** [comment]
> That means I can't compile TimerX on my PC. I want to make the move to other OSes, but just can't find the right time + scared to wreck that computer, it's really precious to me.

1. Why cant you compile it on your PC? We can make blurring exclusive for windows.

2. If you are scared to wreck your pc, just make a backup of it on some hard drive and then do whatever you want. If it goes wrong, just reinstall it (you should know how) and copy over the files from the hard drive.

**Reply by lg-nineoneninethreenine** [reply]
> If you are scared to wreck your pc, just make a backup of it on some hard drive and then do whatever you want. If it goes wrong, just reinstall it (you should know how) and copy over the files from the hard drive.

Issues with this:
1. Windows 7
2. Laptop with only 2 hard drives, with one of them exceeding 90+ GB and the storage limit is 100 GB(OS drive)
3. Linux support seems to be down, it is NOT verified to have Ubuntu LTS-based distros.
4. Don't know if my parents allow me. I know how to flash and install Linux, but I am scared to do it on this. Wondering to switch to a HP or Dynabook computer with Windows 11 for testing and then Zorin OS (maybe Pro) for daily driver usage.

**Reply by not-nef** [reply]
I bet windows 11 can be installed on your pc. Recently i convinced my dad to install it on his PC, which is 14 Years old, and it runs perfectly without any issues (and its fast!!)

**Reply by lg-nineoneninethreenine** [reply]
Bro, it's really old, maybe 5-6 years, meaning no Win11 support. Moreover, I have less storage, no TPM 2.0 or Secure boot. I haven't even seen what the freaking BIOS is. I'll just wait. Google recently acquired Neverware, who made CloudReady. I think you might know what it is. It will make any old computers into Chromebooks without Play Store. There's the new CloudReady 2.0 dubbed Chrome OS Flex and they say it will have Android's Play Store API. It's in development preview so I'll wait for July to install it. I could do it now, but Linux support isn't working, so all good things come to those who wait.
Edit: Flip you Gboard.

**Reply by not-nef** [reply]
> Bro, it's really old, maybe 5-6 years, meaning no Win11 support. Moreover, I have less storage, no TPM 2.0 or Secure boot. I haven't even seen what the freaking BIOS is. I'll just wait. Google recently acquired Neverware, who made CloudReady. I think you might know what it is. It will make any old computers into Chromebooks without Play Store. There's the new CloudReady 2.0 dubbed Chrome OS Flex and they say it will have Android's Play Store API. It's in development preview so I'll wait for July to install it. I could do it now, but Linux support isn't working, so all good things come to those who wait.
> Edit: Flip you Gboard.

Do you think that 14 year old pc had secure boot or a tpm 2 module? Those checks can be bypassed with 2 regkeys!!

**Reply by lg-nineoneninethreenine** [reply]
> Do you think that 14 year old pc had secure boot or a tpm 2 module? Those checks can be bypassed with 2 regkeys!!

You could end up wrecking your PC. Moreover, with the latest PC method of less weight, it's easy for anyone to carry it around. I kinda like Dynabook's approach. Hope to get a nice PC from them or HP!
Moreover, it's illegal and Microsoft is patching up the loopholes in a future update. It's out for insiders, riddled with bugs, but they seem to have enabled a thorough check for it.

**Reply by not-nef** [reply]
ok

**Comment by sumeshir26** [comment]
That'd be reallay cool! We woulod then have a Debian tester on ChromeOS!

**Reply by lg-nineoneninethreenine** [reply]
First, no offense, low expectations please? Crostini doesn't seem to work on the Flex version and I am not interested to spend 25k INR on a Chromebook. When Flex hits stable, I'll try switching over. I don't like Chrome OS much, since all I need is a good desktop OS like Zorin, but it will be kinda a test for now. After that, I'll try getting Zorin Pro or at least get the Core version and customize it to match Zorin OS Pro's Win11 UI.

**Reply by lg-nineoneninethreenine** [reply]
Moreover, Debian is the same everywhere, people just recode it to create new distros.

**Reply by lg-nineoneninethreenine** [reply]
Alright, I am not going to install Chrome OS Flex. Would be happier with Chromium OS or maybe some random Linux distro 😔

**Reply by lg-nineoneninethreenine** [reply]
It doesn't seem to have Android support. Linux has Anbox which supports this natively. Moreover, having Linux as the base is useful for me.

**Comment by sumeshir26** [comment]
And after loonkin at win32micas code, It obly applues mica on windows 11 so Lg can still  compile timerx. Ill impliment this

**Reply by lg-nineoneninethreenine** [reply]
Phew!
